src/lib/REST/Output/ValueObjectVisitor/ApplicationConfigVisitor.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Output\ValueObjectVisitor;
+
+use Ibexa\Contracts\AdminUi\REST\ApplicationConfigRestResolverRegistryInterface;
+use Ibexa\Contracts\Rest\Output\Generator;
+use Ibexa\Contracts\Rest\Output\ValueObjectVisitor;
+use Ibexa\Contracts\Rest\Output\Visitor;
+
+final class ApplicationConfigVisitor extends ValueObjectVisitor
+{
+    private ApplicationConfigRestResolverRegistryInterface $applicationConfigRestResolverRegistry;
+
+    public function __construct(ApplicationConfigRestResolverRegistryInterface $applicationConfigRestResolverRegistry)
+    {
+        $this->applicationConfigRestResolverRegistry = $applicationConfigRestResolverRegistry;
+    }
+
+    /**
+     * @param \Ibexa\AdminUi\REST\Value\ApplicationConfig $data
+     */
+    public function visit(Visitor $visitor, Generator $generator, $data): void
+    {
+        $generator->startObjectElement('ApplicationConfig');
```

Comment:
Each REST outer output visitor should declare its content type. Maybe I don't see it here. What is a content type returned in response headers?

Reply:
For this Visitor it will be: `application/vnd.ibexa.api.ApplicationConfig`
![Screen Shot 2023-12-05 at 12 14 47](https://github.com/ibexa/admin-ui/assets/9850711/e5085122-3be9-4c37-988d-383a9f37d91c)